tests/framework/Optimizers/GeneticAlgorithms/continuous/unconstrained/ZDT1.xml

```diff
@@ -64,7 +64,7 @@
             <mutationProb>1.0</mutationProb>
           </mutation>
         </reproduction>
-        <fitness type="rank_crowding">
+        <fitness type="feasibleFirst">
         </fitness>
         <survivorSelection>rankNcrowdingBased</survivorSelection>
       </GAparams>

```

Comment:
I think this test will still fail. Please revise the opt_export, maybe fitness should be FitnessEvaluation_obj1, FitnessEvaluation_obj2.

Reply:
Fixed!!